src/nsrt_learning/strips_learning/base_strips_learner.py

```diff
@@ -192,7 +192,9 @@ def _recompute_datastores_from_segments(
         for seg_traj in self._segmented_trajs:
             objects = set(seg_traj[0].states[0])
             for segment in seg_traj:
-                segment_option = segment.get_option()
+                segment_option = DummyOption
```

Comment:
wanna do this as an if else like your change to oracle_learner.py?

Reply:
I had that originally, but then getting test coverage was being annoying and led to rabbit holes.. I'll give it another shot though